src/storage/lock_manager/deadlock.rs

```diff
@@ -20,10 +22,13 @@ use std::cell::RefCell;
 use std::fmt::{self, Display, Formatter};
 use std::rc::Rc;
 use std::sync::Arc;
-use std::time::{Duration, Instant};
+use std::time::{Duration, Instant, SystemTime};
 use tokio_core::reactor::Handle;
 use tokio_timer::Interval;
 
+// 5 mins
+const TXN_DETECT_INFO_TTL: u64 = 300000;
```

Comment:
I think 2 minutes is enough, we have a hard limit in TiDB that the ttl is less than one minute.

Reply:
ok